extensions/amp-sidebar/1.0/sidebar.js

```diff
@@ -262,7 +262,9 @@ function SidebarWithRef(
             classes.defaultBackdropStyles
           }`}
           hidden={!side}
-        ></div>
+        >
+          <div className={classes.sidebarChildClass}></div>
```

Comment:
This appears to still work (prevent scrolling on the outer document when the sidebar is open) when I remove this element. See note below about overflow css.